lightning/src/ln/msgs.rs

```diff
@@ -430,6 +430,7 @@ pub(crate) struct UnsignedChannelUpdate {
 	pub(crate) flags: u16,
 	pub(crate) cltv_expiry_delta: u16,
 	pub(crate) htlc_minimum_msat: u64,
+	pub(crate) htlc_maximum_msat: OptionalField<u64>,
```

Comment:
I implemented it, but now fuzzing fails. I believe this is what happens:
1. It generates an UnsignedChannelUpdate with flags=39835 (u16, all flags)
2. It means 155 to store, and 155 as message_flags
3. When deserializing, it can only make message_flags=1, not 155
4. The end-value is (1<<8) | 155, not (155 << 8) | 155

What should we do with this?

Reply:
Ah! Yea, so that's fine, just need to change the fuzz target type, though sadly we may need to define a new one. There are various different rules we apply when fuzzing different messages, you can find them at `fuzz/src/msg_targets/utils.rs` - essentially what we want is `test_msg_hole` but instead of forcing the "hole" to be all-0s, we can just let it be anything (well, really either 0 or 1, but just make it generic). Easiest is likely just replace `test_msg_hole` with a parameter on whether the hole has to be 0 or can be anything.

Reply:
1. I don't see where `test_msg_hole` is forced to be all-0s. It already looks like anything (although the comment says it should be 0s). It just seems to check that everything before and after the hole equals.
2. Said that, I just switched the relevant target to `test_msg_hole 108 1`. Now it passes locally, but something is wrong with CI.

Reply:
Oof. Can you fix the comment there, at least?

Reply:
done!